compiler-plugin/src/main/kotlin/com/google/devtools/ksp/processing/impl/ResolverImpl.kt

```diff
@@ -653,6 +645,13 @@ class ResolverImpl(
             override val arrayType: KSType by lazy { getKSTypeCached(builtIns.array.defaultType.replaceArgumentsWithStarProjections()) }
         }
     }
+
+    internal val mockSerializableType = module.builtIns.numberType.supertypes().single {
```

Comment:
If I understand correctly, this should also work with other builtin types, picking `numberType` is just random, right?

Reply:
Kind of. Number has fewest super types. I tried String as well and it also works.